src/status_im/contexts/chat/messenger/messages/list/view.cljs

```diff
@@ -285,10 +285,15 @@
     (reset! distance-atom new-distance)))
 
 (defn f-messages-list-content
-  [{:keys [insets distance-from-list-top content-height layout-height cover-bg-color distance-atom
+  [{:keys [insets distance-from-list-top content-height layout-height distance-atom
            chat-screen-layout-calculations-complete? chat-list-scroll-y]}]
   (let [theme                    (quo.theme/use-theme-value)
         chat                     (rf/sub [:chats/current-chat-chat-view])
+        community-channel?       (= constants/community-chat-type (:chat-type chat))
+        customization-color      (if community-channel?
+                                   (or (:color chat)
+                                       (rf/sub [:communities/community-color (:community-id chat)]))
+                                   :turquoise)
```

Comment:
@Parveshdhull Does this seem right?

Reply:
I think I may have misunderstood you're earlier comment, but I think you want to resolve customization-color here instead of somewhere else right?